e2e/playwright/snapshot-tests.spec.ts

```diff
@@ -49,7 +49,7 @@ test.setTimeout(60_000)
 
 test(
   'exports of each format should work',
-  { tag: '@snapshot' },
+  { tag: ['@snapshot', '@skipWin', '@skipMacos'] },
   async ({ page, context }) => {
     // skip on macos and windows.
     test.skip(
```

Comment:
Should this skip have been removed?

Reply:
I will not be removing the runtime skips for now since the CI CD pipeline has not been updated. That is what I mean by non destructive stuff.

Only additive changes. I know the github workflows are within this repository but I didn't want to bloat this PR with changing the commands and doing the pipelines in a single PR.